dist/Components/lowerInfo/index.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/extensions */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import OwnerInfo from './ownerInfo.jsx';
+import Rules from './rules.jsx';
+import Cancellation from './cancellation.jsx';
+import Location from './location.jsx';
+
+class LowerInfo extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
Does this need to be a stateful component?

Reply:
I've thought about the same thing, and decided to leave it be so that when we bring all the services together, its an easy addition if we end up needing state anywhere.  I'll remember to eliminate unnecessary ones at the end though